### PR TITLE
Add FAQ

### PR DESCRIPTION
**Pull request by lukekim**

### CODE REVIEW

File: spiceaidocs/docs/faq/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+## 3. How is Spice different?
+
+- **Application-focused:** Designed to integrate at the application level with a flexible 1:1 or 1:N application-to-Spice mapping.
+- **Dual-Engine Acceleration:** Supports OLAP and OLTP databases at the dataset level.
+- **Separation of Materialization and Storage/Compute:** Separates storage and compute for optimal data placement.
+- **Edge to Cloud Native:** Deployable anywhere from standalone instances to Kubernetes containers and public clouds.
+
+## 4. Is Spice a cache?
+
+No, but you can think of Spice data materialization as an active cache or data prefetcher. Unlike a cache that fetches data on a miss, Spice prefetches and materializes filtered data on an interval or as new data is available.
```
Comment (eadgbear) [inline spiceaidocs/docs/faq/index.md:27]: "prefetcher. Unlike" -> "prefetcher, unlike"

File: spiceaidocs/docs/faq/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+- **Proximity:** By co-locating data with applications, Spice reduces latency and improves performance for frequent data access patterns.
+- **Flexibility:** You can quickly spin up multiple, lightweight instances of Spice tailored for different datasets and use cases.
+- **Scalability:** Decentralized materialization allows for better resource control and optimization, as each standalone instance can scale independently based on its workload requirements.
+- **Efficiency:** Reduces the need for massive data movement operations across the network, lowering bandwidth consumption and speeding up query times.
+
+### 7. How does Spice compare to Spark?
+
+Spark is primarily designed for large-scale data processing and batch processing pipelines with its distributed computing engine. In contrast, Spice is focused on accelerating data access and query speeds for applications through materialization and tier-optimized storage strategies.
```
Comment (eadgbear) [inline spiceaidocs/docs/faq/index.md]: NP: "batch-processing"

File: spiceaidocs/docs/faq/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+No, but you can think of Spice data materialization as an active cache or data prefetcher. Unlike a cache that fetches data on a miss, Spice prefetches and materializes filtered data on an interval or as new data is available.
+
+## 5. Is Spice a CDN for databases?
+
+Yes, Spice acts like a CDN for different data sources. It allows you to load a working set of your database where it's most frequently accessed, such as from a data application or for AI inference.
+
+## 6. How does Spice differ from Trino/Presto and Dremio?
+
+Spice is tailored for data and AI applications, while systems like Trino/Presto and Dremio are optimized for big data and real-time analytics. Spice specializes in high-concurrency, low-latency access and materialization of data close to the application.
```
Comment (eadgbear) [inline spiceaidocs/docs/faq/index.md]: "materialization of data" -> "data materialization" for simplicity

**Review by eadgbear — DISMISSED** [review]

File: spiceaidocs/docs/faq/index.md (hunk 3 above)
Comment (lukekim) [inline spiceaidocs/docs/faq/index.md]:
```suggestion
Spice is designed for data and AI applications, while systems like Trino/Presto and Dremio are optimized for big data and real-time analytics. Spice specializes in high-concurrency, low-latency access and data materialization close to the application.
```

File: spiceaidocs/docs/faq/index.md (hunk 2 above)
Comment (lukekim) [inline spiceaidocs/docs/faq/index.md]:
```suggestion
Spark is primarily designed for large-scale data processing and batch-processing pipelines with its distributed computing engine. In contrast, Spice is focused on accelerating data access and query speeds for applications through materialization and tier-optimized storage strategies.
```

**Review by Sevenannn — APPROVED** [review]

**Pull request merged**